src/components/Numbers.js

```diff
@@ -1,25 +1,62 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
+import calculate from '../logic/calculate';
```

Comment:
- [ ] Kindly note that your linters are very important they help you structure well your code and prevent future bugs. Kindly solve them instead of disabling them.